airflow/www/static/js/grid/components/Table.jsx

```diff
@@ -187,28 +187,30 @@ const Table = ({
           })}
         </Tbody>
       </ChakraTable>
-      {totalEntries > data.length && (
+      {(canPreviousPage || canNextPage) && (
```

Comment:
Found and fixed a bug here right?

Reply:
Exactly, this wouldn't have worked with any client-side pagination.